src/daft-parquet/src/read.rs

```diff
@@ -460,9 +461,9 @@ async fn stream_parquet_single(
 }
 
 #[allow(clippy::too_many_arguments)]
-async fn read_parquet_single_into_arrow(
+async fn read_parquet_single_into_arrow<T: AsRef<str>>(
```

Comment:
should this instead be ToString, since we call that right after?

Reply:
Removed the generic here and just pass around `String`s